js/src/3rdparty/sms-verification/index.js

```diff
@@ -15,17 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { stringify } from 'querystring';
-import React from 'react';
-
-export const termsOfService = (
```

Comment:
Suppose on #3, we could go with something like markdown - will actually kill 2 birds with one stone - 

- be readable as text
- be transformable into pretty HTML